<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstRunWizard</class>
 <widget class="QDialog" name="FirstRunWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Last.fm Desktop App</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="welcome">
     <property name="text">
      <string notr="true">Welcome</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string notr="true">Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="LoginPage" name="loginPage"/>
     <widget class="AccessPage" name="accessPage"/>
     <widget class="PluginsPage" name="pluginsPage"/>
     <widget class="PluginsInstallPage" name="pluginsInstallPage"/>
     <widget class="BootstrapPage" name="bootstrapPage"/>
     <widget class="BootstrapProgressPage" name="bootstrapProgressPage"/>
     <widget class="TourScrobblesPage" name="tourScrobblesPage"/>
     <widget class="TourMetadataPage" name="tourMetadataPage"/>
     <widget class="TourRadioPage" name="tourRadioPage"/>
     <widget class="TourFinishPage" name="tourFinishPage"/>
     <widget class="TourLocationPage" name="tourLocationPage"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="margin">
      <number>40</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="importIcon">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="importLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="back">
       <property name="text">
        <string notr="true">Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skip">
       <property name="text">
        <string notr="true">Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="custom">
       <property name="text">
        <string notr="true">Custom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="text">
        <string notr="true">Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finish">
       <property name="text">
        <string notr="true">Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/LoginPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AccessPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/AccessPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PluginsPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/PluginsPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PluginsInstallPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/PluginsInstallPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BootstrapPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/BootstrapPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TourScrobblesPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/TourScrobblesPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TourMetadataPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/TourMetadataPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TourRadioPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/TourRadioPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TourFinishPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/TourFinishPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TourLocationPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/TourLocationPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BootstrapProgressPage</class>
   <extends>QWidget</extends>
   <header>../Wizard/BootstrapProgressPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
